### DISCUSSION TITLE
buddhism/2023/lotus_sutra/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#81] [General]
# buddhism/2023/lotus_sutra/

https://shanenull.com/buddhism/2023/lotus_sutra/

**Comment by shane0** [comment]
reading playlist

https://youtube.com/playlist?list=PL0w7f776wBnEQhYM6pGE1FDVUUTE2xrj6&si=76369FvIOViSbLDU

**Comment by shane0** [comment]
# chapters

| Chapter | Title                  | Number Meaning Topics                   |
|---------|------------------------|------------------------------------------|
| 1       | Introduction           | -                                          |
| 2       | Expedient Means        | -                                          |
| 3       | Simile and Parable     | -                                          |
| 4       | Belief and Understanding| -                                         |
| 5       | The Parable of the Medicinal Herbs | Healing, Wholeness, Balance       |
| 6       | Bestowal of Prophecy   | Prophecy, Enlightenment                 |
| 7       | The Parable of the Phantom City | Illusion, Reality, Truth        |
| 8       | Prophecy of Enlightenment for Five Hundred Disciples | Spiritual Growth, Guidance |
| 9       | Prophecy of Enlightenment for Bodhisattvas | Bodhisattva Path, Compassion |
| 10      | The Teacher of the Law | Dharma, Teaching, Guidance               |
| 11      | The Emergence of the Treasure Tower | Manifestation, Revelation     |
| 12      | Devadatta                | Overcoming Obstacles, Transformation    |
| 13      | Encouragement for Keeping the Sutra | Perseverance, Commitment       |
| 14      | Peaceful Practices      | Inner Peace, Harmony                    |
| 15      | Emerging from the Earth | Bodhisattva Vows, Liberation            |
| 16      | The Lifespan of the Tathagata | Eternal Truth, Transcendence   |
| 17      | Distinction of Benefits | Karmic Merit, Virtue                    |
| 18      | The Benefits of Joyful Acceptance | Joy, Contentment                |
| 19      | The Benefits of the Teacher of the Law | Dharma Benefits, Guidance  |
| 20      | The Bodhisattva Never Disparaging | Respect, Non-discrimination        |

**Comment by shane0** [comment]
# 32 characteristics of a Buddha

also known as the "32 Marks of the Great Man" or "32 Signs of a Great Being," are physical and spiritual attributes that are said to be present in a fully enlightened Buddha. These characteristics are commonly mentioned in Buddhist scriptures, including the Pali Canon and Mahayana texts.

1. **Flat and Level Feet:** The soles of his feet are flat and level.
2. **Thousand-Spoked Wheel Mark:** The wheels of his chariot leave imprints with a thousand spokes.
3. **Golden-Colored Skin:** His skin has a golden hue.
4. **Fine, Soft Hair:** His hair is fine and curls to the right.
5. **Ankle-Length Arms:** His arms extend down to his knees.
6. **Webbed Fingers and Toes:** His fingers and toes are webbed.
7. **Full, Rounded Heels:** The heels of his feet are full and rounded.
8. **Thighs Like a Royal Stag:** His thighs are like those of a royal stag.
9. **Hands Reaching Below the Knees:** His hands reach below his knees.
10. **Well-Set Male Organ:** His male organ is well-retracted and covered by a sheath.
11. **A Golden Color in His Body:** His body has a golden color.
12. **Ten-Foot Aura:** He has an aura that extends ten feet around him.
13. **Soft and Tender Skin:** His skin is soft and tender.
14. **A Body Like a Banyan Tree:** His body is like a banyan tree, with a broad, rounded top.
15. **A Clear and Melodious Voice:** His voice is clear and melodious.
16. **Blue Eyes:** His eyes are blue.
17. **Eyelashes Like a Cow:** His eyelashes are like those of a cow.
18. **White Curled Hair:** His hair is white and curls to the right.
19. **Soft and Curled Hair:** His hair is soft and curls naturally.
20. **White Teeth:** His teeth are white and evenly spaced.
21. **A Large Tongue:** His tongue is large and covers his mouth when not speaking.
22. **Braided Hair:** His hair is braided.
23. **Upright and Well-Supported Neck:** His neck is upright and well-supported.
24. **Gentle Breeze-Like Breath:** His breath is like a gentle breeze.
25. **Fine Hair Between the Eyebrows:** There is fine hair between his eyebrows.
26. **Wide, Deer-Like Eyes:** His eyes are wide and like those of a deer.
27. **Radiant Forehead:** His forehead is radiant.
28. **Noble Nose:** His nose is noble and straight.
29. **Prominent Shoulders:** His shoulders are prominent.
30. **Forty Teeth:** He has forty teeth.
31. **A Web of Lines on His Palms:** There is a web of lines on his palms.
32. **Long, Beautiful Fingers:** His fingers are long and beautiful.

**Reply by shane0** [reply]
page 33 diamond sutra
https://www.ctworld.org.tw/Buddhist%20e-Books/Books01/index.html

the thathagata cannot be recognized by means of his 32 characteristics

**Comment by shane0** [comment]
Playlist

https://youtube.com/playlist?list=PLxF0FhhTjZ2-lavJ97aVKFuLwipV4580B&si=xcxgrjgA4WQAKu3r

**Comment by shane0** [comment]
https://www.youtube.com/playlist?list=PLFb9wBs0vR0euw2b8VJzNtvBl9x8obLw7

**Comment by shane0** [comment]
https://www.bdk.or.jp/document/dgtl-dl/dBET_T0262_LotusSutra_2007.pdf

**Comment by shane0** [comment]
https://github.com/shane0/buddhism/issues/91

**Comment by shane0** [comment]
english playlist
https://www.youtube.com/watch?v=t1_sKzjXivw&list=PLFb9wBs0vR0ddWzKeQbPg2bFm5ZzAMZDR
